themes/bigspring/layouts/partials/home/adopters.html

```diff
@@ -0,0 +1,14 @@
+{{ if site.Data.homepage.adopters.enable }}
+{{ with site.Data.homepage.adopters }}
+<section class="width-80 vertical-align-center padding-top-down-50">
+    <h2 class="text-center padding-top-down-10">Trusted and Adopted by</h2>
+    <div id="adopters" class="flex column-mobile">
+    {{ range .logos }}
+      <div class="img-container-md vertical-align-center">
```

Comment:
For end user logos, please use: NASA, Bosch, IBM, Zeiss, ZScaler, Alibaba Cloud, Rakuten, PwC, Fujitsu, Vonage

Reply:
Can also add Lufthansa @marcduiker